plog/plog

```diff
@@ -358,9 +369,9 @@ compress() {
 
     info Compressing log file...
     if [[ -n "$DRY" ]]; then
-        >&2 echo "gzip --best -c $1 >  $out"
+        >&2 echo "$compress_progam --best -c $1 >  $out"
     else
-        safe_pv "$1" | gzip --best > "$out"
+        safe_pv "$1" | "$compress_progam" --best > "$out"
```

Comment:
Typo:

    - progam 
    + program

Reply:
Thanks. The latest push has been tested.